metadata-ingestion/src/datahub/ingestion/source/looker/looker_file_loader.py

```diff
@@ -86,7 +91,17 @@ def _load_viewfile(
             self.viewfile_cache[path] = looker_viewfile
             return looker_viewfile
         except Exception as e:
-            self.reporter.failure("Failed to parse lkml file", path, exc=e)
+            self.reporter.failure(
+                title="LKML File Parsing Error",
+                message=f"Failed to parse lookml file {path}",
+                exc=e,
+                log=True,
+            )
+
+            logger.debug(f"Raw file content for path {path}")
+
+            logger.debug(raw_file_content)
```

Comment:
make this one debug log message

Reply:
They are already debug.

Reply:
src/datahub/ingestion/source/dbt/dbt_cloud.py